uv_link_t.gyp

```diff
@@ -1,7 +1,7 @@
 {
   "variables": {
     "gypkg_deps": [
-      "git://github.com/libuv/libuv.git@^1.9.0 => uv.gyp:libuv",
+      "git://github.com/libuv/libuv.git@^1.12.0 => uv.gyp:libuv",
```

Comment:
Strange... I think it should automatically pick `1.12.0`, because `^1.9.0` allows this. Why is it needed?